client/src/components/profile/components/HeatMap.js

```diff
@@ -57,18 +68,30 @@ function HeatMap({ calendar, streak }) {
       <FullWidthRow>
         <CalendarHeatMap
           classForValue={value => {
-            if (!value) {
+            if (value.count < 1) {
               return 'colour-empty';
             }
             if (value.count > 4) {
               return 'colour-scale-a-lot';
             }
             return `colour-scale-${value.count}`;
           }}
-          endDate={today}
-          startDate={sixMonthsAgo}
+          endDate={endOfCalendar}
+          startDate={startOfCalendar}
+          tooltipDataAttrs={value => {
+            let valueCount = '';
+            if (value.count === 1) {
```

Comment:
```suggestion
            if (value && value.count === 1) {
```
More defensive coding